endtoend/evaluators/validator.go

```diff
@@ -46,6 +49,13 @@ func afterNthEpoch(afterEpoch uint64) func(uint64) bool {
 	}
 }
 
+// Not including first epoch because of issues with genesis.
+func betweenEpochs(fromEpoch uint64, toEpoch uint64) func(uint64) bool {
```

Comment:
please change to boolean type name

```suggestion
func isBetweenEpochs(fromEpoch uint64, toEpoch uint64) func(uint64) bool {
```